html.html.markdown

```diff
@@ -0,0 +1,84 @@
+---
+language: html
+filename: learnhtml.html
+contributors:
+  - ["Alois de Gouvello", "https://github.com/aloisdg"]
+---
+
+HTML, for HyperText Markup Language, is a declarative language used for creating and visually representing a webpage. Web browsers can read HTML files and render them into visible or audible web pages.
+ 
+HTML is a subset of Standard Generalized Markup Language (SGML) and is specified by the World Wide Web Consortium (W3C).
+
+```html
+
+<!DOCTYPE html>
+
+<!--  Comments are between arrows. -->
+
+<!-- <!DOCTYPE> informs the browser which version of HTML you used.
+	 You cannot write valid HTML before it. -->
+	 
+<!--  HTML is written in the form of nested HTML elements consisting of
+      tags enclosed in angle brackets : <tag>. A lot of tags exist.
+      We cannot describe all of them. -->
+
+<!--  Always start with the tag <html>. It represents the root of your page. -->
+<html>
+
+	<!--  An HTML page is divide in two singles parts.
+    		<head> provides general information about the document.
+    		<body> represents the content of an HTML document. -->
+  <head>
+      <!-- We write in utf-8.  -->
+    <meta charset="utf-8" />
+
+    <title>Here we write the Document title</title>
+  </head>
+  <body>
+  	<!-- Here we write the Document content -->
+  	<!-- HTML is a semantic langage. Use the rightest possible element. -->
+
+		<!-- <div> is a basic block element. -->
+    <div>
+    <!--  There are 6 levels of heading. <h1> is the largest
+          and h6 is the smallest). -->
+      <h1>Main heading in my document</h1>
+  
+      <p>This is a paragraph.</p>
+  
+			<!-- HTML supports attributes. -->
+			<!-- id marks a single element. -->
```

Comment:
I added some informations. I think it is still a bit light. I dont want to overload the comment.